src/main/resources/static/css/cas.css

```diff
@@ -100,15 +103,27 @@ aside, section, main {
 }
 
 .mdc-drawer-app-content {
-    flex: auto;
-    overflow: auto;
-    position: relative;
+    /* flex: auto; */
+    /* overflow: auto; */
+    /* position: relative; */
+    flex-direction: column;
+
 }
 
 .main-content {
     display: flex;
 }
 
+.main-background-gradient {
+    background: linear-gradient(90deg, rgba(255, 255, 255, 0.75), rgba(255, 255, 255, 0.80)), url(/images/dark-blue-gradient.png) center center no-repeat;
```

Comment:
Follow-up: I ran into an issue that this `/images/dark-blue-gradient.png` is not found. Not sure if this has anything to do of button having a white background.